### DISCUSSION TITLE
Mise à jour pour x-touch

### DISCUSSION THREAD

**Discussion by danimusicos** [bjoluc/cubase-mcu-midiremote#39] [General]
Bonjour, une nouvelle version du script pour Cubase 13 sera-t-elle disponible ? Actuellement, seule la version 1.90 est disponible, mais elle n'est malheureusement pas optimisée pour Cubase 13. Merci.
Hello, will a new version of the script for Cubase 13 be available? Currently only version 1.90 is available, but unfortunately it is not optimized for Cubase 13. Thank you.

**Comment by bjoluc** [comment]
Hey, v1.10.0 simplifies button and foot pedal assignments in Cubase 13. Nothing else CB13-related I can do, sadly.
